release_notes.md

```diff
@@ -3,17 +3,6 @@
 <!-- Please add your release notes in the following format:
 - My change description (#PR)
 -->
-- Update Java Worker Version to [2.8.1](https://github.com/Azure/azure-functions-java-worker/releases/tag/2.8.1)
-- Update Python Worker Version to [4.9.0](https://github.com/Azure/azure-functions-python-worker/releases/tag/4.9.0)
-- Track ApplicationInsights ingestion service response by subscribing to transmission event [PR](https://github.com/Azure/azure-functions-host/pull/8828)
-- Update ApplicationInsights packages to 2.21.0 (#8838)
-- Support model-type bindings for out-of-proc workers (#8815)
-- Fix the duplicate log issue with Java ApplicationInsights agent
-- Update Node.js Worker Version to [3.5.2](https://github.com/Azure/azure-functions-nodejs-worker/releases/tag/v3.5.2)
-- Making feature flag for worker indexing opt-out instead of opt-in (#8864)
-- Update PowerShell Worker 7.2 to 4.0.2496 [Release Note](https://github.com/Azure/azure-functions-powershell-worker/releases/tag/v4.0.2496)
-
-
-**Release sprint:** Sprint 132
-[ [bugs](https://github.com/Azure/azure-functions-host/issues?q=is%3Aissue+milestone%3A%22Functions+Sprint+132%22+label%3Abug+is%3Aclosed) | [features](https://github.com/Azure/azure-functions-host/issues?q=is%3Aissue+milestone%3A%22Functions+Sprint+132%22+label%3Afeature+is%3Aclosed) ]
+**Release sprint:** Sprint 133
+[ [bugs](https://github.com/Azure/azure-functions-host/issues?q=is%3Aissue+milestone%3A%22Functions+Sprint+133%22+label%3Abug+is%3Aclosed) | [features](https://github.com/Azure/azure-functions-host/issues?q=is%3Aissue+milestone%3A%22Functions+Sprint+133%22+label%3Afeature+is%3Aclosed) ]
 - Update App Service Authentication/Authorization on Linux Consumption from 1.4.17 to 1.5.1 [Release Note](https://github.com/Azure/app-service-announcements/issues/406)
```

Comment:
I know that you didn't do this but do you mind moving this to be above the release sprint section?